_data/wcag-act-rules/act-implementations.yml

```diff
@@ -0,0 +1,66 @@
+- name: Alfa
+  vendor: Siteimprove
+  uniqueKey: alfa
+  type: Automated tool
+  standards:
+    - WCAG 2.1 Level A - AAA
```

Comment:
I am not clear what this means. A, AA, and AAA, A and AAA? We may need to discuss how to name conformance levels